doc/profiler.rst

```diff
@@ -49,14 +50,14 @@ You can increase the number of levels displayed for a particular object by click
 
 |
 
-You can click the dropdown or press the :kbd:`Down Arrow` key in the filename field to recall paths of previous profiled files.
+By clicking the dropdown or press the :kbd:`Down Arrow` key in the filename field, you can recall paths of previous profiled files.
 
 .. image:: images/profiler/profiler-previous-paths.png
    :alt: Spyder Profiler pane, showing dropdown of previous profiled files
 
 |
 
-You can save the data for a given run to disk as a file with the ``.Result`` extension using the :guilabel:`Save data` button, and it can be loaded to compare with another run using the :guilabel:`Load data` button.
+Finally, you can save the data for a given run to disk as a file with the ``.Result`` extension using the :guilabel:`Save data` button. This can be loaded to compare with a previous run of the same file using the :guilabel:`Load data` button.
```

Comment:
```suggestion
Finally, you can save the data for a given run to disk as a file with the ``.Result`` extension using the :guilabel:`Save data` button.
This can be loaded to compare with a previous run of the same file using the :guilabel:`Load data` button.
```

Line break.